src/test/resources/features/actions/line_mover.feature

```diff
@@ -1,54 +1,78 @@
+    <rule_like2> NAME2:
         <rule_section2>: <content2>
-    <rule_like1> <name1>:
-        <rule_section1>:
-            <content1>
+    <rule_like1> NAME1:
+        <rule_section1>: <content1>
     """
     Examples:
-      | rule_like1  | rule_like2  | rule_section1  | rule_section2  | content1    | name1 | name2   | content2   |
-      | rule        | rule        | input          | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
-      | checkpoint  | rule        | input          | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
-      | subworkflow | rule        | workdir        | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
-      | rule        | checkpoint  | input          | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
-      | checkpoint  | checkpoint  | input          | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
-      | subworkflow | checkpoint  | workdir        | output         | "file.txt"  | NAME1 | NAME2   | "file.txt" |
-      | rule        | subworkflow | input          | workdir        | "file.txt"  | NAME1 | NAME2   | "file.txt" |
-      | checkpoint  | subworkflow | input          | workdir        | "file.txt"  | NAME1 | NAME2   | "file.txt" |
-      | subworkflow | subworkflow | workdir        | workdir        | "file.txt"  | NAME1 | NAME2   | "file.txt" |
-      | rule        | def         | input          | n              | "file.txt"  | NAME1 | NAME2() | int        |
-      | checkpoint  | def         | input          | n              | "file.txt"  | NAME1 | NAME2() | int        |
-      | subworkflow | def         | workdir        | n              | "file.txt"  | NAME1 | NAME2() | int        |
+      | rule_like1  | rule_like2  | rule_section1  | rule_section2  | content1       | content2      |
+      | rule        | rule        | input          | output         | "file1.txt"    | "file2.txt"   |
+      | checkpoint  | rule        | input          | output         | "file1.txt"    | "file2.txt"   |
+      | subworkflow | rule        | workdir        | output         | "file1.txt"    | "file2.txt"   |
+      | rule        | checkpoint  | input          | output         | "file1.txt"    | "file2.txt"   |
+      | checkpoint  | checkpoint  | input          | output         | "file1.txt"    | "file2.txt"   |
+      | subworkflow | checkpoint  | workdir        | output         | "file1.txt"    | "file2.txt"   |
+      | rule        | subworkflow | input          | workdir        | "file1.txt"    | "file2.txt"   |
+      | checkpoint  | subworkflow | input          | workdir        | "file1.txt"    | "file2.txt"   |
+      | subworkflow | subworkflow | workdir        | workdir        | "file1.txt"    | "file2.txt"   |
+      | rule        | def         | input          | n              | "file1.txt"    | int           |
+      | checkpoint  | def         | input          | n              | "file1.txt"    | int           |
```

Comment:
`def` isn't `rule like` snakemake section

Reply:
I added different test for python functions